src/interactive/Toaster.jsx

```diff
@@ -0,0 +1,136 @@
+import CSSTransitionGroup from 'react-addons-css-transition-group';
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+
+export const DELAY_TIME = 3000;
+
+/**
+ * @module Toaster
+ */
+class Toaster extends React.PureComponent {
+	constructor(props) {
+		super(props);
+
+		this.timeouts = [];
+		this.dismissedToasts = [];
+
+		this.cloneToast = this.cloneToast.bind(this);
+		this.setDismissedToast = this.setDismissedToast.bind(this);
+		this.clearTimeouts = this.clearTimeouts.bind(this);
+		this.setTimer = this.setTimer.bind(this);
+
+		this.state = {
+			toasts: this.props.toasts.map(this.cloneToast)
+		};
+	}
+
+	/**
+	 * @param {Object} dismissedToast - `Toast` components to remove from state
+	 * @returns undefined
+	 */
+	setDismissedToast(dismissedToast) {
```

Comment:
nitpick - I recommend renaming this to `dismissToast` - it's not 'setting' anything